tests/foreman/cli/test_docker.py

```diff
@@ -998,18 +1157,9 @@ class DockerComputeResourceTestCase(CLITestCase):
     def setUpClass(cls):
         """Create an organization and product which can be re-used in tests."""
         super(DockerComputeResourceTestCase, cls).setUpClass()
-        cls.org_id = entities.Organization().create_json()['id']
+        cls.org = make_org()
 
-    @stubbed()
-    @run_only_on('sat')
-    @data(
-        gen_string('alpha'),
-        gen_string('alphanumeric'),
-        gen_string('numeric'),
-        gen_string('latin1'),
-        gen_string('utf8'),
-        gen_string('html'),
-    )
+    @data(*valid_data_list())
```

Comment:
I think it's fine to use itertools in a test. itertools is in the standard library, and we use it elsewhere, so it's not as if you're pulling in some exotic dependency.

I wouldn't stress too much about making making the code in the `@data(...)` decorator look nice. We're going to drop DDT, and when we do that, that'll be a better time to be inventive and come up with good ways to gernerate data.